rewards/aws/helpers.py

```diff
@@ -7,14 +7,14 @@
 
 logger = logging.getLogger("aws-helpers")
 
-if config("TEST", "False").lower() in ["true", "1", "t", "y", "yes"]:
+if config("KUBE", "True").lower() in ["true", "1", "t", "y", "yes"]:
```

Comment:
What's this?

Reply:
Its an environment variable that detects whether or not we are in the kube environment